### PR TITLE
Ivory 307 user provides name and email

### PR DESCRIPTION
**Pull request by duncannixon**
IVORY 307 - User provides name and email

Includes use of new form-layout.html and form validation method.

Unit tests to follow after

### CODE REVIEW

**Review by Jozzey — CHANGES_REQUESTED** [review]
Excellent work, all looks very good.
Just one minor thing, could you add a link to the Home page for the new "Who owns the item" page for easy access until the application is more whole.

**Review by dgretho — APPROVED** [review]

File: server/routes/who-owns-the-item.route.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    const errors = _validateForm(payload)
+
+    if (errors.length) {
+      return h
+        .view(Views.WHO_OWNS_ITEM, {
+          ...buildErrorSummary(errors)
+        })
+        .code(400)
+    } else {
```
[the remark this answers is not in the document]
Comment (duncannixon) [reply server/routes/who-owns-the-item.route.js:23]: Fair enough, fixed.

File: server/routes/user-details/owner/contact-details.route.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    if (errors.length) {
+      return h
+        .view(Views.CONTACT_DETAILS, {
+          title: _getTitle(ownerApplicant),
+          ownerApplicant: ownerApplicant === Options.YES,
+          ..._getContext(request),
+          ...buildErrorSummary(errors)
+        })
+        .code(400)
+    } else {
```
[the remark this answers is not in the document]
Comment (duncannixon) [reply server/routes/user-details/owner/contact-details.route.js:39]: Fair enough, fixed.

**Comment by duncannixon** [comment]
Added a link to the Home page for the new "Who owns the item" page

**Review by Jozzey — APPROVED** [review]
All good, thanks.

**Pull request merged**